api/src/wfl/sink.clj

```diff
@@ -102,7 +102,7 @@
   (str/join " " ["Found additional attributes in fromOutputs that are not"
                  "present in the entityType."]))
 
-(defn ^:private terra-workspace-validate-request-or-throw
+(defn terra-workspace-validate-request-or-throw
```

Comment:
Huh, I'm surprised that this is the only validation taking place on a sink request.